core/src/main/resources/jenkins/model/Jenkins/configure.jelly

```diff
@@ -60,17 +60,9 @@ THE SOFTWARE.
           <st:include page="${descriptor.globalConfigPage}" from="${descriptor}" />
         </f:rowSet>
       </j:forEach>
-
-      <l:hasAdministerOrManage>
-        <f:bottomButtonBar>
-          <f:submit value="${%Save}" />
-          <f:apply />
-        </f:bottomButtonBar>
-      </l:hasAdministerOrManage>
+      <j:set var="readOnlyMode" value="${!h.hasPermission(app.MANAGE)}"/>
```

Comment:
Needed because of https://github.com/jenkinsci/jenkins/pull/9813/files#diff-04f3f5f17eaeb24ac6c8f50acae68a658010e18d2de16105942d34e151542fbfR57